mlflow/models/evaluation/base.py

```diff
@@ -2058,6 +2060,16 @@ def pred_sample(eval_df, _builtin_metrics, _artifacts_dir):
             error_code=INVALID_PARAMETER_VALUE,
         )
 
+    model_predict_func = None
+    if model is not None and hasattr(model, "predict") and callable(model.predict):
+        # Store reference to original predict method
+        predict_func = model.predict
+
+        def model_predict_func(x):
+            # Disable all autologging except for traces
```

Comment:
This comment `Disable all autologging except for traces` should more naturally go to the function `restrict_langchain_autologging_to_traces_only` itself. Right now I don't think this comment adds much over the name of the function. However, some discussion in that doc string around care taken to ensure non-langchain use cases are not broken would be nice